packages/app/src/pages/dashboard/migrate.tsx

```diff
@@ -0,0 +1,314 @@
+				},
+				{
+					key: 'balance-staked',
+					title: t('dashboard.stake.portfolio.balance.staked'),
+					value: truncateNumbers(stakedKwentaBalanceV2, 2),
+					onClick: () => setCurrentTab(StakeTab.Escrow),
+				},
+			],
+		},
+		{
+			category: t('dashboard.stake.portfolio.rewards.title'),
+			card: [
+				{
+					key: 'rewards-claimable',
+					title: t('dashboard.stake.portfolio.rewards.staking'),
+					value: truncateNumbers(claimableBalanceV2, 2),
+					onClick: () => setCurrentTab(StakeTab.Staking),
+				},
+				{
+					key: 'rewards-trading',
+					title: t('dashboard.stake.portfolio.rewards.trading'),
+					value: truncateNumbers(kwentaRewards, 4),
+					onClick: () => setCurrentTab(StakeTab.Staking),
+				},
+			],
+		},
+		{
+			category: t('dashboard.stake.portfolio.escrow.title-v2'),
+			card: [
+				{
+					key: 'escrow-staked',
+					title: t('dashboard.stake.portfolio.escrow.staked'),
+					value: truncateNumbers(stakedEscrowedKwentaBalanceV2, 2),
+					onClick: () => setCurrentTab(StakeTab.Escrow),
+				},
+				{
+					key: 'escrow-vestable',
+					title: t('dashboard.stake.portfolio.escrow.vestable'),
+					value: truncateNumbers(totalVestableV2, 2),
+					onClick: () => setCurrentTab(StakeTab.Escrow),
+				},
+			],
+		},
+		{
+			category: t('dashboard.stake.portfolio.escrow.title-v1'),
+			card: [
+				{
+					key: 'escrow-staked',
+					title: t('dashboard.stake.portfolio.escrow.staked'),
+					value: truncateNumbers(stakedEscrowedKwentaBalance, 2),
+					onClick: () => setCurrentTab(StakeTab.Escrow),
+				},
+				{
+					key: 'escrow-vestable',
+					title: t('dashboard.stake.portfolio.escrow.vestable'),
+					value: truncateNumbers(totalVestable, 2),
+					onClick: () => setCurrentTab(StakeTab.Escrow),
+				},
+			],
+		},
+	]
+
+	return (
+		<>
+			<Head>
+				<title>{t('dashboard-stake.page-title')}</title>
+			</Head>
+			<StakingHeading>
+				<FlexDivCol rowGap="5px">
+					<StyledHeading variant="h4">Migrate to Staking V2</StyledHeading>
+					<Body color="secondary">
+						Lorem ipsum dolor sit amet consectetur. Ut in nisl ut quam condimentum lacus.
+					</Body>
+				</FlexDivCol>
+				<StyledButton
+					size="xsmall"
+					isRounded
+					textTransform="none"
+					onClick={() => window.open(EXTERNAL_LINKS.Docs.Staking, '_blank')}
+				>
+					Docs →
+				</StyledButton>
+			</StakingHeading>
+			<StepsContainer columnGap="15px">
+				{MIGRATE_STEPS.map(({ key, copy, label, value, buttonLabel, active, onClick }, i) => (
+					<StyledStakingCard key={key} $active={active}>
+						<StyledHeading variant="h4">Step {i + 1}</StyledHeading>
+						<Body size="small" color="secondary">
+							{copy}
+						</Body>
+						<Spacer height={25} />
+						<FlexDivRowCentered>
+							<FlexDivCol rowGap="5px">
+								<Body size="small" color="secondary">
+									{label}
+								</Body>
+								<Body size="large" color="preview">
+									{value}
+								</Body>
+							</FlexDivCol>
+							<Button
```

Comment:
Can we add disabled / loading

Reply:
Yes. That's true. The button state should be same as the `Step`.